rewrite-java-test/src/test/java/org/openrewrite/java/style/AutodetectTest.java

```diff
@@ -1125,4 +1127,379 @@ boolean count(String... strings) {
               .isEqualTo(3);
         }
     }
+
+    @Nested
+    class AnnotationWrapping {
+        @Test
+        void classLevelAnnotations() {
+            var cus = jp().parse(
+              """
+                import java.lang.annotation.Repeatable;
+                
+                @Repeatable(Foo.Foos.class)
+                @interface Foo {
+                    @interface Foos {
+                        Foo[] value();
+                    }
+                }
+                
+                @Foo
+                @Foo
+                class Test {
```

Comment:
Shouldn't have this test next to interface and normal class also have an `enum` and `record` line of code here? At jvm level, those are all classes.